assets/app/view/game/buy_sell_shares.rb

```diff
@@ -30,15 +30,17 @@ def render
           end
 
           # Put up one buy button for each buyable percentage share type in market.
-          # In case there are more than one type of percentages in market or if shares are not 10%
-          # (e.g. 18MEX), show percentage type on button.
-          pool_shares
+          # In case there are more than one type of percentages in market or if shares are not the
+          # standard percent (e.g. 5% in 18MEX), show percentage type on button.
+          # Do skip president's share in case there are other shares available.
+          buyables = pool_shares
```

Comment:
one statement buyable = pool_shares.select do ...

Reply:
I think it needs to be done in two steps, as I use the size of pool_shares (line 38) and buyable (line 40).